docs/v3.x/guides/custom-admin.md

```diff
@@ -29,7 +29,7 @@ Let's eject this file to be able to customize it.
 
 In this new file, paste the current [HomePage container](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin/src/containers/HomePage/index.js) code.
 
-To run your application, you will have to run the `yarn develop --watch-admin` command.
+To run your application, you will have to run the `yarn develop --watch-admin` / `npm run develop -- --watch-admin` command.
```

Comment:
@meganelacheny is it better to use the tabs here?

Reply:
Yes, @henrych4 could you please use tabs to display properly the different commands?
It should be something like that:

` To run your application, run the following command: `

` :::: tabs `

` ::: tab yarn ` 

` ```bash `
` yarn develop --watch-admin `
` ``` `

` ::: ` 

` ::: tab npm `

` ```bash `
` npm run develop -- --watch-admin ` 
` ``` ` 

` ::: ` 

` :::: `

Reply:
Updated